02-first_steps.Rmd

```diff
@@ -240,6 +240,175 @@ adsl %>%
   kable_classic_2(full_width = F)
 ```
 
+### Handling of missing values
+
+Missing values in SAS and R:
+
+* SAS 
+  + Missing value is a blank/a single decimal point for character/numeric variables, see details in [Missing Values in SAS](https://documentation.sas.com/doc/en/pgmsascdc/9.4_3.5/lrcon/p175x77t7k6kggn1io94yedqagl3.htm);
+  
+* R
+  + Missing data in R appears as `NA`. `NA` is not a string nor a numeric value, but an indicator of missingness;
+  + `NA` and " " are different in R: " " is a blank string, while `NA` is missing;
```

Comment:
By the way, there is another difference.

In SAS trailing spaces are removed when comparing strings while this is not done in R. For example,
`" " = "'` is `1` in SAS but `" " == ""` is `FALSE` in R.

Thus I would use `""` for blank strings in R.

Reply:
Thanks, @bundfussr ! those are valuable feedback. please let me know if further updates are needed :)